src/components/ebay-carousel/browser.json

```diff
@@ -1,9 +1,23 @@
 {
     "dependencies": [
+        "@ebay/skin/utility",
         "@ebay/skin/less",
         "@ebay/skin/iconfont",
         "@ebay/skin/dist/icon/ds4/icon.css",
```

Comment:
Since this component is explicitly referencing the ds4 icons it can break the ds6 icon styles (in my case the close button). We should update this to use ds6 icons.